Test_iDRAC_driver_WS-Man_inspection.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+TEST_CASE_NAME="Test iDRAC driver WS-Man inspection (idrac-wsman)"
+
+if [ $# -ne 1 ]; then
+    echo
+    echo "WARNING: Usage: Test_iDRAC_driver_WS-Man_inspection (idrac-wsman).sh <NODE_UUID>"
+    echo
+    exit 1
+fi
+
+NODE_UUID=$1
+
+echo "INFO: NODE UUID is: $NODE_UUID"
+
+. ~/devstack/openrc admin
+
+set -x
+
+echo "INFO: Checking node existance"
+result=$(openstack baremetal node show $NODE_UUID -c 'uuid' -f value)
+if [ -z "$result" ]
+then
+        echo "ERRRO: Node is not present or provided invalid node uuid "
+        exit 1
+else
+        echo "INFO: Preparing node for inspection"
+fi
+
+echo "INFO: Checking node provision_state"
+state=$(openstack baremetal node show $NODE_UUID -c provision_state -f value)
+if [ $state == 'manageable' ]
+then
+        echo "INFO: Node provision_state is ${state}"
+        #start_inspection=$(openstack baremetal node inspect $NODE_UUID)
+else
+	echo "ERROR: Invalid Node provision_state it should be manageable"
+	exit 1
+fi
+echo "INFO: Checking node inspect_interface"
+current_interface=$(openstack baremetal node show $NODE_UUID -c "inspect_interface" -f value)
+if [ $current_interface == 'idrac-wsman' ]
+then
+        echo "INFO: Node inspect_interface is ${current_interface}"
+else
+        echo "ERROR: Invalid Node inspect_interface"
+        exit 1
+fi
+#todo use while loop and add inspect failed condition 
+
+echo "INFO: Starting ws-man inspection"
+openstack baremetal node inspect $NODE_UUID
+
+while :
+do
+	inspection_result=$(openstack baremetal node show $NODE_UUID -c provision_state -f value)
+	if [ $inspection_result == 'manageable' ]
+	then
+		echo "INFO: iDRAC driver WS-Man inspection successfully"
+		break
+	elif [ $inspection_result == "inspect failed" ]
+	then
+		echo "ERROR: Node inspection failed "
```

Comment:
Add the last_error msg from 'openstack baremetal node show node-uui | grep last_err' command.

Reply:
done